src/Raven.Server/Documents/Indexes/IndexField.cs

```diff
@@ -108,5 +110,10 @@ public IndexFieldOptions ToIndexFieldOptions()
                 TermVector = TermVector
             };
         }
+
+        public static string GetAnalyzedAutoIndexFieldName(string name)
+        {
+            return $"Analyzed({name})";
```

Comment:
We have two options here: either match the indexing option - `Analyzed` or the querying function - `search`. I've chosen the first one since it's the name of the field stored in the index according to the indexing options but `search(name)` also works for me. 

What do you think about renaming indexing options: `Analyzed` -> `Search`, `NotAnalyzed` -> `Exact` to match querying terminology?